vmdb/db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/Methods.class/__methods__/catalogbundleinitialization.rb

```diff
@@ -109,17 +145,13 @@ def tag_parent_service(service, options_hash)
   dialog_options_hash = {}
 
   # Set all dialog options pertaining to the catalog item plus any options destined for the catalog bundle
-  unless options_hash[0].nil?
-    unless options_hash[group_idx].nil?
-      # Merge child options with global options if any
-      dialog_options_hash = options_hash[0].merge(options_hash[group_idx])
-    else
-      dialog_options_hash = options_hash[0]
-    end
-  else # unless options_hash[0].nil?
-    unless options_hash[group_idx].nil?
-      dialog_options_hash = options_hash[group_idx]
-    end
+  if options_hash[0]
+    dialog_options_hash = options_hash[group_idx] unless options_hash[group_idx].nil?
+  elsif options_hash[group_idx]
+    dialog_options_hash = options_hash[0]
+  else
+    # Merge child options with global options if any
+    dialog_options_hash = options_hash[0].merge(options_hash[group_idx])
   end
```

Comment:
Oh no it's the same...I didn't see that you flipped the unless .nil?

On that note should like 149 end with `if options_hash[group_idx]` instead of the unless .nil?